FLIR/conservator/generated/schema.py

```diff
@@ -77,6 +77,10 @@ class FavoriteAssetType(sgqlc.types.Enum):
 Float = sgqlc.types.Float
 
 
+class GraphqlID(sgqlc.types.Scalar):
+    __schema__ = schema
+
+
```

Comment:
Did the generator script add this?  There are a lot of changes from ID -> GraphqlID that I don't see in Conservator's schema JSON.

Reply:
Unless this schema is coming from your branch?